File: Two_Sum/two_sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i=0;i<nums.size();i++){
            for(int j=i+1; j<nums.size();j++){
                int tmp;
                tmp = nums[i]+nums[j];
                if(target == tmp){
                    nums.clear();
                    nums.push_back(i);
                    nums.push_back(j);
                }
            }
        }
        return nums;
    }
};

File: Two_Sum/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        save = {}
        for i in range(len(nums)):
            num = target - nums[i]
            if(num in save):
                return [save[num], i]
            else:
                save[nums[i]] = i;
